content/logs/faq/how-to-send-logs-to-datadog-via-external-log-shippers.md

```diff
@@ -140,9 +137,19 @@ This approach can be especially useful for sending to Datadog logs that have hei
     ```
     $template DatadogFormat,"YOURAPIKEY <%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% - - [metas ddsource=\"mysourcename\"] %msg%\n"
     ```
-    Do not forge to replace mysourcename by the appropriate value.
+    Do not forge to replace mysourcename by the appropriate value.  
+
+9. (Optional) Datadog cuts inactive connections after a period of inactivity.  
+    For some reason, there are Rsyslog versions that are not able to reconnect properly when necessary. To mitigate this issue we propose to use time markers so the connection never stops. Add the following 2 lines in your Rsyslog configuration:  
```

Comment:
Instead of saying `For some reason ...`, I would say: `Some Rsyslog versions are not able to reconnect properly when necessary.`.

Reply:
Instead of saying: 

```To mitigate this issue we propose to use time markers so the connection never stops. Add the following 2 lines in your Rsyslog configuration: ```

I would say

```To mitigate this issue, you may use time markers so the connection never stops. To achieve this, add the following 2 lines in your Rsyslog configuration: ```

Reply:
👍